PWGCF/EbyEFluctuations/Tasks/CMakeLists.txt

```diff
@@ -34,8 +34,8 @@ o2physics_add_dpl_workflow(antiproton-cumulants-mc
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::PWGCFCore
                     COMPONENT_NAME Analysis)
 		
-o2physics_add_dpl_workflow(identified-meanpt-fluctuations
-                    SOURCES IdentifiedMeanPtFluctuations.cxx
+o2physics_add_dpl_workflow(event-mean-pt-id
+                    SOURCES eventMeanPtId.cxx
```

Comment:
Here you have removed one file `IdentifiedMeanPtFluctuations.cxx` and one executable `identified-meanpt-fluctuations` from the build of `O2Physics`